### ISSUE TITLE
Disable node hover by attributes

### ISSUE THREAD

**Issue by havardl** [jacomyal/sigma.js#1033]
Hi all,

I have a fairly large graph with 22K nodes and 70K edges, so I'm trying to find ways of optimizing the rendering. By default, it seems that sigma.js hovers all nodes. I know I can turn off label hovering, but is it possible to disable node hovering (color change and edge connections) in general? 

If so, can I specify which types of nodes which should not have a hover based on their attributes and would this even effect the rendering?

**Comment by Yomguithereal** [comment]
Hello @havardl. There is a global setting to disable hovering which is called `enableHovering`. I don't recall an attribute to do this on a per-node basis though.

**Comment by havardl** [comment]
Hi @Yomguithereal, thanks for this.

Yes, I've set that to false which disables labels on hover, but still when you hover a node there are events being fired as the graph then shows how the node is connected to other nodes and dims unconnected nodes. My idea was that if I disabled this function, then the graph would be more responsive when interacting with it.

If it's not possible to disable specific nodes, then another alternative could be to disable any kind of hovering and then only use click events for the nodes. But then I'm back to to question if it's at all possible to disable any kind of hovering.
